docker/Dockerfile.mipsel-unknown-linux-uclibc

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:20.04
```

Comment:
Renaming this to `Dockerfile.mipsel-unknown-linux-uclibc-cross` would be make this consistent with the rest of the [repository](https://github.com/cross-rs/cross-toolchains/tree/main/docker).

Reply:
This would then change the image name to `ghcr.io/cross-rs/mipsel-unknown-linux-uclibc-cross` rather than `ghcr.io/cross-rs/mipsel-unknown-linux-uclibc` (with the local tag here).